trulens_eval/trulens_eval/feedback/provider/openai.py

```diff
@@ -13,12 +14,13 @@
 logger = logging.getLogger(__name__)
 
 
-class OpenAI(Provider):
+class OpenAI(LLMProvider):
     """Out of the box feedback functions calling OpenAI APIs.
```

Comment:
Also add to each of the inheritors

See available feedback functions in the base [LLMProvider](https://www.trulens.org/trulens_eval/api/feedback/#trulens_eval.trulens_eval.feedback.provider.%3CCHANGEME%3E.LLMProvider)

Reply:
I added this into new provider api references